one-app-statics/postinstall.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2019 American Express Travel Related Services Company, Inc.
```

Comment:
This should be 2020.

Reply:
I want to remain in 2019! 2020 has been too harsh so far!